### ISSUE TITLE
Broken with ligatures

### ISSUE THREAD

**Issue by leshow** [sainnhe/icursive-nerd-font#2]
I don't think the font plays very well with ligatures, this is Fira Code with Operator Mono and ligatures enabled in vscode

![Screenshot from 2020-05-11 14-51-55](https://user-images.githubusercontent.com/1128302/81600220-424e6480-9397-11ea-9d99-b36fdb1914ec.png)

**Comment by leshow** [comment]
Ah, it looks like the font doesn't have any Fira Code in it at all. almost all of that looks like operator mono.

**Comment by sainnhe** [comment]
I'm unable to fix this, help wanted.

**Comment by ImtiazKabir** [comment]
@leshow I am facing this exact  same problem. Just wondering if you found some kind of "fix".

**Comment by dlejay** [comment]
I am experiencing the same problem on macOS.
I understand Fira Code iCursive Op is your preferred font choice @sainnhe, do you use ligatures?
Is the problem related to the platform?

**Comment by sainnhe** [comment]
Yes, I use it sometimes. Now my favorite one is Lilex iCursive S12 and the ligatures works nicely.

It seems like this problem only happens with some fonts like FiraCode Op, I have no idea what's going on.

**Comment by dlejay** [comment]
Strangely, Lilex iCursive S12 does not give my cursive italics on macOS whereas FiraCode Op does.
